crates/symbolicator-sources/src/index.rs

```diff
@@ -0,0 +1,104 @@
+use std::collections::BTreeSet;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader, Write};
+use std::path::Path;
+use std::sync::Arc;
+
+#[derive(Debug, Clone, Default)]
+pub struct SymstoreIndex {
+    files: BTreeSet<Arc<str>>,
+}
+
+impl SymstoreIndex {
```

Comment:
Docs here would be great, also linking to the upstream docs, etc. https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/symstore-transactions